ext/zend_test/iterators.c

```diff
@@ -0,0 +1,123 @@
+/*
+  +----------------------------------------------------------------------+
+  | Copyright (c) The PHP Group                                          |
+  +----------------------------------------------------------------------+
+  | This source file is subject to version 3.01 of the PHP license,      |
+  | that is bundled with this package in the file LICENSE, and is        |
+  | available through the world-wide-web at the following url:           |
+  | https://www.php.net/license/3_01.txt                                 |
+  | If you did not receive a copy of the PHP license and are unable to   |
+  | obtain it through the world-wide-web, please send a note to          |
+  | license@php.net so we can mail you a copy immediately.               |
+  +----------------------------------------------------------------------+
+  | Authors: Aaron Piotrowski <aaron@trowski.com>                         |
```

Comment:
Author name needs fixing :)

Reply:
Yeah, I just copy-pasted the license block as you can see...

Feel free to suggest a change!

Reply:
@ju1ius If you don't want to credit yourself you can just leave it out.

Reply:
Ok, I don't really care about claiming authorship of a one-line patch and a unit test, and that information would become inaccurate as soon as anyone else contributes, so I'm just going to leave it out.